docs/visual-basic/language-reference/error-messages/bc36645.md

```diff
@@ -1,47 +0,0 @@
----
```

Comment:
@Youssef1313 recently added this file (to replace BC32051) in #19928, however, the error message is already covered by https://github.com/dotnet/docs/blob/main/docs/visual-basic/misc/bc36648-bc36645.md.